### PR TITLE
Small fix for certified by, wrong validation

### PR DESCRIPTION
**Pull request by seeker25**

### CODE REVIEW

**Review by Jxio — APPROVED** [review]

**Review by ritvick — APPROVED** [review]

File: auth-web/package-lock.json [hunk 1]
```diff
@@ -36792,7 +36792,7 @@
     },
     "fas-ui": {
       "version": "git+ssh://git@github.com/bcgov/fas-ui.git#67e5772573db41e49c8d55b371aaeebba8622d11",
-      "from": "fas-ui@git+https://github.com/bcgov/fas-ui.git#v1.0.6",
+      "from": "fas-ui@github:bcgov/fas-ui#v1.0.6",
```
Comment (severinbeauvais) [inline auth-web/package-lock.json:36795]: Weird.

**Review by severinbeauvais — APPROVED** [review]
Looks OK to me but I'm not clear what the problem was or how this fixes it...?

**Comment by seeker25** [comment]
`const validCertifier = (this.isGovStaffAccount && !this.authorizationName)`
always makes  `get isFormValid (): boolean` return false

EX. this.authorizationName = 'Blaze, Johnny'

**Comment by severinbeauvais** [comment]
> `const validCertifier = (this.isGovStaffAccount && !this.authorizationName)` always makes `get isFormValid (): boolean` return false

Does this form still work when either reg user or staff is trying to add a firm and certify?

**Pull request merged**